src/mjolnir/graphtilebuilder.cc

```diff
@@ -149,6 +149,8 @@ GraphTileBuilder::GraphTileBuilder(const std::string& tile_dir,
     EdgeInfoBuilder eib;
     eib.set_wayid(ei.wayid());
     eib.set_mean_elevation(ei.mean_elevation());
+    eib.set_bike_network(ei.bike_network());
+    eib.set_speed_limit(ei.speed_limit());
```

Comment:
ive been suffering from this for a long time but havent had a chance to look. thank you so much!

Reply:
So you knew there was a bug? I just stumbled across it as I debugging a bike route and noted that it was always saying the EdgeInfo had bike network values set (but the directed edge did not). We don't really use the values from EdgeInfo (just the flag in directed edge for costing) so it didn't seem like a big deal - but any data error makes me curious!